routers/routes/routes.go

```diff
@@ -523,6 +523,11 @@ func RegisterRoutes(m *macaron.Macaron) {
 				return
 			}
 		})
+		m.Group("/branches", func() {
```

Comment:
@lunny do we really need new unit for this? I think that unit code should be for all code tabs (code, commit and branches), what do you think?